Source/Engine/Syntax/ConjunctionSyntax.cs

```diff
@@ -3,11 +3,8 @@
 // Licensed under the Apache License, Version 2.0.
 //--------------------------------------------------------------------------------------------------
 
-using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
-using System.Linq;
-using System.Text;
```

Comment:
Please, don't remove usings that seem useless, they allow better code completion (using namespace don't create assembly dependency).